conans/cli/commands/remote.py

```diff
@@ -0,0 +1,103 @@
+import json
+
+from conans.cli.command import conan_command, conan_subcommand, OnceArgument
+
+
+def output_remote_list_json(info, out):
+    myjson = json.dumps(info, indent=4)
+    out.writeln(myjson)
+
+
+def output_remote_list_cli(info, out):
+    for remote_info in info:
+        output_str = "{}: {} [Verify SSL: {}, Enabled: {}]".format(remote_info["name"],
+                                                                   remote_info["url"],
+                                                                   remote_info["verify"],
+                                                                   remote_info["enabled"])
+        out.writeln(output_str)
+
+
+@conan_subcommand(formatters={"cli": output_remote_list_cli, "json": output_remote_list_json})
+def remote_list(*args, conan_api, parser, subparser):
+    """
+    List current remotes
+    """
+    args = parser.parse_args(*args)
+    info = [{"name": "remote1", "url": "https://someurl1", "verify": True, "enabled": False},
+            {"name": "remote2", "url": "https://someurl2", "verify": False, "enabled": True},
+            {"name": "remote3", "url": "https://someurl3", "verify": True, "enabled": True},
+            {"name": "remote4", "url": "https://someurl4", "verify": False, "enabled": False}]
+    return info
+
+
+@conan_subcommand()
+def remote_add(*args, conan_api, parser, subparser):
+    """
+    Add a remote
+    """
+    subparser.add_argument("remote", help="Name of the remote to add")
+    subparser.add_argument("url", help="Url for the rempote")
+    subparser.add_argument("-v", "--verify-ssl", action=OnceArgument, default="True",
```

Comment:
Suggest using a different short arg here, since "-v" is usually short for "verbose" or "version".

Reply:
Maybe go a bit further? What if we don't define short args as default, and only add them for a few, very used and verbose cases? (Like ``--settings`` => ``-s``) What do you think @czoido ?

Reply:
Thanks a lot for reporting this @cobalt77. Yes, I totally agree with @memsharded, let's just use for very common things like `-h --help`, `-s -settings`, `-r --remote` ...